test/externalTests/ens.sh

```diff
@@ -28,22 +28,23 @@ verify_input "$@"
 BINARY_TYPE="$1"
 BINARY_PATH="$2"
 
-function compile_fn { npx truffle compile; }
-function test_fn { npm run test; }
+function compile_fn { yarn build; }
+function test_fn { yarn test; }
 
 function ens_test
 {
-    local repo="https://github.com/ensdomains/ens.git"
+    local repo="https://github.com/ensdomains/ens-contracts.git"
     local branch=master
-    local config_file="truffle.js"
+    local config_file="hardhat.config.js"
 
-    local compile_only_presets=()
+    local compile_only_presets=(
+        legacy-no-optimize        # Compiles but tests fail to deploy GovernorCompatibilityBravo (code too large).
+    )
```

Comment:
In this case it's really because I don't want to spend time on tweaking their test/ganache config for this to pass and the this particular config preset is not all that important for us anyway - we skip it in many tests due to stack-related errors. I'd prioritize it more if it was the only preset that could work or if it was the `legacy-optimize-evm+yul` one.